src/config.json

```diff
@@ -993,9 +993,11 @@
     "metadusk.io",
     "metabase.finance",
     "metamarc.io",
-    "metapass.fyi"
+    "metapass.fyi",
+    "roco.finance",
```

Comment:
You have a trailing comma on the end of that last line in the whitelist object; this is a syntax error in the JSON and why the tests aren't passing.